charts/piraeus/templates/operator-issuer.yaml

```diff
@@ -0,0 +1,39 @@
+{{- $fullName := include "operator.fullname" . -}}
+{{- if or (eq .Values.linstorHttpsMethod "cert-manager") (eq .Values.linstorSslMethod "cert-manager") }}
```

Comment:
Is it usual for charts to create their own issuer? I was under the impression that issuers are set up once by the admin?

Reply:
I suppose you're talking about kind ClusterIssuers which are mainly used to issue certificates using let's encrypt service.

Since we're using self-signed certificate and own creating own Authority, then we have to create our own issuer, this issuer is namespace scoped and nobody else could use it in the cluster.